### DISCUSSION TITLE
samgeo not working with only one bounding box

### DISCUSSION THREAD

**Discussion by jborderon13** [opengeos/segment-geospatial#183] [Q&A] [answered]
Hi, 

First thank you for you work ! 

It seems that samgeo doesnt work with only one bounding box it gives me the following error when I run sam.predict(boxes=boxes, point_crs="EPSG:4326", output="mask.tif", dtype="uint8")  :

AttributeError: 'numpy.ndarray' object has no attribute 'clone' 

If I add an other bounding box it works perfectly.

Do you know why ? And what can I do to use this model on image where sometimes we only have one bb ?

Thanks !

**Comment by giswqs** [comment]
There are multiple sam models in the package. Please provide compelete source code for debugging.

**Comment by jborderon13** [comment]
I'm using the notebook which is on google colab (https://colab.research.google.com/github/opengeos/segment-geospatial/blob/main/docs/examples/box_prompts.ipynb ) with :

 sam = SamGeo(
    model_type="vit_h",
    automatic=False,
    sam_kwargs=None,
)

**Comment by giswqs** [comment, marked as answer]
This bug has been fixed in #184. Update it using `pip install git+https://github.com/opengeos/segment-geospatial.git`
